cmd/codegen/app/generate_clientset.go

```diff
@@ -83,7 +83,7 @@ func NewGenerateClientSetCmd(genOpts GenerateOptions) *cobra.Command {
 	cobraCmd.Flags().StringVarP(&o.InputPackage, optionInputPackage, "i", "", "Input package, must specify")
 	cobraCmd.Flags().StringVarP(&o.OutputPackage, optionOutputPackage, "o", "", "Output package, must specify")
 	cobraCmd.Flags().StringVarP(&o.InputBase, optionInputBase, "", wd, "Input base, defaults working directory")
-
+	cobraCmd.Flags().BoolVarP(&o.Global, global, "", false, "use the users GOPATH")
```

Comment:
Why not make the default `true`?

Reply:
Because it doesn't function properly except in some edge cases (you have none of the code already checked out in your `GOPATH`, or if you do, `master` is checked out).

Reply:
Would have been nice with an issue related to this providing more context.